example/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [project]
-name = "connecpy-example"
+name = "example"
```

Comment:
It seems uv had some issue with this name not matching the folder name when adding the new project.

Reply:
I don't think this affects this repo too badly but I filed an issue anyways to see what could cause it

https://github.com/astral-sh/uv/issues/14861

Reply:
But somehow it's working now...